db/migrate/20150202094307_add_range_to_trend_counters.rb

```diff
@@ -0,0 +1,22 @@
+class AddRangeToTrendCounters < ActiveRecord::Migration
+  def change
+    add_column :trend_counters, :interval_start, :datetime
+    add_column :trend_counters, :interval_end, :datetime
```

Comment:
Number of records that are fetched should be relatively small, it shouldn't cross 1000 records. I don't think it justifies an index